### PR TITLE
Improvements to admin interface

### PR DESCRIPTION
**Pull request by anuschka**
Addressing enhancements in #145.
Removed projects and added picture to UserProfile.
Added project - UserProfile relationship.

### CODE REVIEW

**Comment by anuschka** [comment]
I need some guideliness (pleeease) how to add UserProfile - user relationship.

**Comment by mojcas** [comment]
Maybe something like here, under customizing user?https://docs.djangoproject.com/en/dev/topics/auth/#storing-additional-information-about-users
In templates it is for example used as userprofile.user.first_name

**Comment by anuschka** [comment]
Thanks @mojcas. I am trying to work it out using http://stackoverflow.com/questions/9879687/adding-a-manytomanywidget-to-the-reverse-of-a-manytomanyfield-in-the-django-admi

File: api/models.py [hunk 1]
```diff
@@ -21,6 +21,9 @@ class UserProfile(models.Model):
 
 	#Relations
 	user = models.OneToOneField(User) 
+	def __unicode__(self):
+		return u'%s' % (self.user.username)
```
Comment (ialja) [inline api/models.py]: You're awesome for adding the `__unicode__` method! :+1: 

Although, you could simplify this line by writing just `return self.user.username`(same for the return under the Projects model). Using string replacements (the `%s` part) makes sense if you have more variables, but not when you have just one.

**Comment by ialja** [comment]
@anuschka Can you please sync your fork with upstream?

**Comment by anuschka** [comment]
@ialja Will do but first have to resolve :
>CONFLICT (content): Merge conflict in api/models.py
>Auto-merging api/admin.py
>CONFLICT (content): Merge conflict in api/admin.py
>Automatic merge failed; fix conflicts and then commit the result.

**Comment by anuschka** [comment]
I will close this one and create a new PR.

**Comment by goranche** [comment]
> I will close this one and create a new PR

@anuschka Why??

**Comment by anuschka** [comment]
@goranche It was not clean with all the files I adopted synching :smile:

**Comment by goranche** [comment]
@anuschka we've discussed this and would like you to fix this PR... I've reopened it an will be closing #185

**Comment by ialja** [comment]
Nice job! :+1: Would also be nice to get the `__unicode__` method for Task and Event model (it would make a nice small PR ;) )

**Comment by anuschka** [comment]
Will do :hand:

**Pull request merged**